frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -16,7 +16,7 @@ This includes handling the urls and view state
 */
 
 const SHOW_TIMEOUT_MESSAGE_AFTER = 15000
-const SHOW_TIMEOUT_MESSAGE_FUNNELS = 30000
+const SHOW_TIMEOUT_MESSAGE_FUNNELS = 3000
```

Comment:
Was this accidental or on purpose? 

30sec is quite long though. AFAIK users start to get distracted and leave after ~10sec, so 15might still work, but anything more is too long. 3sec might be too short though.